pre_cache_action.sh

```diff
@@ -32,6 +32,19 @@ input_packages="${@:6}"
 
 # Trim commas, excess spaces, and sort.
 log "Normalizing package list..."
+
+# Ensure APT package lists are updated if stale (for nektos/act compatibility)
+# Uses the same logic as install_and_cache_pkgs.sh
+if [[ -z "$(find -H /var/lib/apt/lists -maxdepth 0 -mmin -5 2>/dev/null)" ]]; then
+  log "APT package lists are stale, updating..."
+  if command -v apt-fast > /dev/null 2>&1; then
```

Comment:
Will this ever happen? apt-fast should have a guarantee of being installed

Reply:
This will slow down the action though. The point is that this part run without needing to update. Is there a way to detect if the nektos/act is being ran?